mlflow/server/js/src/experiment-tracking/components/ParallelCoordinatesPlotPanel.js

```diff
@@ -25,36 +25,126 @@ export class ParallelCoordinatesPlotPanel extends React.Component {
     sharedParamKeys: PropTypes.arrayOf(String).isRequired,
     // An array of metric keys shared by all runs
     sharedMetricKeys: PropTypes.arrayOf(String).isRequired,
+    // An array of different parameter keys across runs
+    diffParamKeys: PropTypes.arrayOf(String).isRequired,
   };
 
-  state = {
-    // Default to select all parameters
-    selectedParamKeys: this.props.sharedParamKeys,
-    // Default to select the first metric key.
-    // Note that there will be no color scaling if no metric is selected.
-    selectedMetricKeys: this.props.sharedMetricKeys.slice(0, 1),
+  constructor(props) {
```

Comment:
Coolio! Can we make "Select Diff Only" the default, _if_ there are non-constant keys?

Reply:
I think we can. I'll add this change.